modules/kubernetes/ingress/main.tf

```diff
@@ -0,0 +1,65 @@
+terraform {
+  required_version = "~> 1.3"
+
+  required_providers {
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
+      version = "2.19.0"
+    }
+  }
+
+  backend "azurerm" {}
+}
+
+provider "kubernetes" {
+  config_path = var.config_path
+}
+
+resource "kubernetes_ingress_v1" "ingress" {
+  count = var.enabled ? 1 : 0
```

Comment:
Waarom is er een enabled var?
Een disabled ingress is toch gelijk aan geen ingress toevoegen?

Reply:
Ja, maar als dat per omgeving verschilt kan je dat niet met Terragrunt doen